dags.yaml

```diff
@@ -1570,3 +1570,23 @@ bqetl_braze_currents:
     retry_delay: 30m
   tags:
     - impact/tier_2
+
+bqetl_pageload_v1:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - wichan@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
+    end_date: null
+    owner: wichan@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: '2023-10-01'
```

Comment:
I read this as intention to have the DAG automatically create tasks since this date to "backfill" the table. Is this correct? 

If `catchup` setting for the DAG is not set to TRUE will this correctly create tasks for each of the schedule dates since the start_date until today?

Alternatively, I'd say this to a more recent date and manually backfill the table until the most recent date (could probably use managed backfills here too?) and change the start_date to current date.

Reply:
So catchup flag on the DAG means that the scheduler will look at the start_date and the schedule_interval and calculate all execution dates when the DAG should have run and create DAG instances for each of them and run the tasks (think @submission_date is the same as the execution_date in most of our cases).

If the setting is turned off (which if I recall is the case with the autogenerated DAGs), it would only figure out what is the most recent execution_date that should have run for the DAG and only run that one date and continue scheduling future dates as normal.